starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+h1 a{
+	color: #ffb782;
+	letter-spacing: 2px;
+	font-weight: 500;
+	text-decoration: none;
+	font-size: 48px;
+	
+}
+
+body {
+	font-family: Lato;
+	padding: 30px 925px 0px 50px;
+}
+
+nav a{
+	font-size: 22px;
+	color: #c4c4c4;
+	padding: 0px 5px 0px 5px;
+}
+
+h2 {
+	font-weight: 500;
+	font-style: bold;

```

Comment:
Bold is an invalid value for font-style. You are basically making this h2 bold by changing the font-weight in the line above.